Pipeline/CodeCoverageReport/README.md

```diff
@@ -0,0 +1,36 @@
+# Code Coverage Report
+
+This script can be used to extract and print information collected by the IBM Debug Code Coverage feature.
```

Comment:
This scrips extracts and prints test coverage information of an automated test execution in a pipeline collected ...